.github/workflows/test.yml

```diff
@@ -84,6 +67,11 @@ jobs:
           # workflow only change but user not authorized
           exit 1
 
+      # BUG: This task attempts to run the `full` behave tag if the PR under
+      # test is a release from dev to prod, but the matcher condition that would
+      # emit this appears broken. Investigate the setting of the
+      # charts_release_branch output, or just run smoke tests and remove the
+      # condition associated with this output.
```

Comment:
Should we create an issue in our repo to track this ?

I'm making some sense out of this comment, but capturing the full context right now would be beneficial.